prism/prism.js

```diff
@@ -0,0 +1,757 @@
+/**
+ * Copyright 2021 The Rode Authors
```

Comment:
I don't think we want our copyright header in this one

Reply:
I removed our copyright but now the license check is failing.. Looks like there [isn't a way to ignore files](https://github.com/google/addlicense/issues/53).

Reply:
Looks like someone opened a [PR](https://github.com/google/addlicense/pull/73) to add the ability to ignore.  For now I'm going to add our copyright header back in so I can keep moving on. I'll make an issue to address this in the future.